unittests/finalizer_update_tests.cpp

```diff
@@ -71,6 +71,18 @@ BOOST_AUTO_TEST_CASE(savanna_set_finalizer_multiple_test) { try {
    size_t num_keys    = 50u;
    size_t finset_size = 21u;
 
+   auto verify_block_finality_generation = [](const signed_block_ptr& block, uint32_t gen, const bls_public_key& key) {
```

Comment:
Good check! 

No big deal, but I think this is a more accurate name.

```suggestion
   auto verify_block_finality_policy_diff = [](const signed_block_ptr& block, uint32_t gen, const bls_public_key& key) {
```